### ISSUE TITLE
launching errors

### ISSUE THREAD

**Issue by technovangelist** [davidgs/link-maker#12]
## Prerequisites

- [ ] Using npm
- [ ] Using an up-to-date [`main` branch](https://github.com/davidgs/link-maker/tree/main)
- [ ] Using latest version of devtools. [Check the docs for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools/)
- [ ] For issue in production release, add devtools output of `DEBUG_PROD=true npm run build && npm start`

grabbed the install from releases

## Current Behavior

when installing the arm dmg, i get a blank screen. when installing from the zip that says it is for mac, i get this: 
![CleanShot 2023-04-05 at 09 16 10](https://user-images.githubusercontent.com/633681/230142110-14f96717-f7fc-40dc-9dfe-fbbdf13f8c40.png)

## Steps to Reproduce

1.

2.

3.

4.

## Your Environment

- Node version :
- electron-react-boilerplate version or branch :
- Operating System and version :
- Link to your project :

**Comment by davidgs** [comment]
Sorry, I am aware of this and am trying to solve it. If I build from source locally, it runs fine. If I run `yarn run package` locally (with my Apple certs) it build proper `.dmg` files which contain valid, working apps. When GitHub builds them via the GitHub action, they come out with a blank screen.

**Comment by technovangelist** [comment]
yup, `yarn run package` runs and the installers it creates work fine.

**Comment by davidgs** [comment]
Yeah, and I can't for the life of me figure out why the actions build is broken. ¯\_(ツ)_/¯ Still investigating.
